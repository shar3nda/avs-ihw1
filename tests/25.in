3 9 1 4 3 0 7 3 2 -10