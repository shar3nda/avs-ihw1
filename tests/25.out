0 7 3 2 -10